### PR TITLE
Number Accuracy

### PR DESCRIPTION
**Pull request by LilyLME**
It is true Tor will be translated to many more languages in the future, but I just thought the numbers should be up to date. This PR might be irrelevant, but I just wanted to propose it.

### CODE REVIEW

**Review by emmapeel2 — APPROVED** [review]
Yes, I agree that it is good to stay updated. Thanks for your attention to detail!

I wonder if the solution for this outdated string is maybe to remove such information, or edit the line in a way that we don't need to edit often? Because we keep adding languages all the time, so this string is bound to be outdated again.

Other solution will be to add the update of this page to the release process but I think that would be overkill.

Maybe we should have a little script that maintains the number of languages? I will ask arthuredelstein how does he counts the published languages in https://torpat.ch/locales

**Comment by LilyLME** [comment]
> Yes, I agree that it is good to stay updated. Thanks for your attention to detail!
> 
> I wonder if the solution for this outdated string is maybe to remove such information, or edit the line in a way that we don't need to edit often? Because we keep adding languages all the time, so this string is bound to be outdated again.
> 
> Other solution will be to add the update of this page to the release process but I think that would be overkill.
> 
> Maybe we should have a little script that maintains the number of languages? I will ask arthuredelstein how does he counts the published languages in https://torpat.ch/locales

Okay @emmapeel2 . I will like to get updates. On my end,i will do some research and try to propose an automatic counter.

File: content/tbb/tbb-37/contents.lr [hunk 1]
```diff
@@ -5,6 +5,6 @@ title: I need Tor Browser in a language that's not English.
 seo_slug: need-tor-browser-language-not-english
 ---
 description:
-We want everyone to be able to enjoy Tor Browser in their own language. Tor Browser is now [available in 25 different languages](https://www.torproject.org/download/languages/), and we are working to add more. Want to help us translate? [See here](https://tb-manual.torproject.org/becoming-tor-translator/)
+We want everyone to be able to enjoy Tor Browser in their own language. Tor Browser is now [available in 30 different languages](https://www.torproject.org/download/languages/), and we are working to add more. Want to help us translate? [See here](https://tb-manual.torproject.org/becoming-tor-translator/)
```
Comment (gusgustavo) [inline content/tbb/tbb-37/contents.lr]: Use this url: https://community.torproject.org/localization/becoming-tor-translator/

Comment (gusgustavo) [reply content/tbb/tbb-37/contents.lr]: And as a suggestion, replace "See here" to something more meaningful like "Want to help us translate? [Become a Tor translator](https://community.torproject.org/localization/becoming-tor-translator/)!"

**Comment by LilyLME** [comment]
Edited as proposed @gusgustavo .
https://github.com/torproject/support/pull/76/files

**Comment by enjeck** [comment]
@LilyLME It appears the number has changed yet again. Now, it's 32 languages, I believe. https://blog.torproject.org/new-release-tor-browser-90 :smile: Maybe this warrants a PR?

**Pull request merged**